tests/helpers.py

```diff
@@ -46,6 +46,7 @@
 Redis Environment
 """
 testenv["redis_host"] = os.environ.get("REDIS_HOST", "127.0.0.1")
+testenv["redis_db"] = os.environ.get("REDIS_DB", "0")
```

Comment:
For consistency can we follow the same as [here](https://github.com/instana/python-sensor/pull/608/files#diff-7fb11d01efb2e6788562533f26902ed0563b68bfbd51ccecf879ec3ad594f869L41) and [here](https://github.com/instana/python-sensor/pull/608/files#diff-7fb11d01efb2e6788562533f26902ed0563b68bfbd51ccecf879ec3ad594f869R32)?

```suggestion
testenv["redis_db"] = os.environ.get("REDIS_DB", "instana_test_db")
```

Reply:
Redis dbs are identified by integer indexes.
https://stackoverflow.com/a/35932972/14637536

Reply:
Ah interesting, I just checked the redis documentation. In that case, I guess it would be intuitive to pass db an integer value and we can check if it is not None explicitly.

Reply:
You are correct, passing db as an integer seems to be correct way to do it.